src/navigator_data_ingest/base/types.py

```diff
@@ -80,6 +80,7 @@ class Document(BaseModel):
     slug: str  # Unique identifier created by backend
     publication_ts: datetime
     source_url: Optional[AnyHttpUrl]  # Original source URL
+    download_url: Optional[AnyHttpUrl]  # Cached document URL
```

Comment:
will this actually be an HTTP URL? I thought we were either supplying an s3 URL or object path?

Reply:
Seeing as we use requests to download the document my assumption was that we would pass in an s3 object url or cdn domain / path to object to facilitate download. 

"download_url": "https://cpr-data-bucket.s3.eu-west-1.amazonaws.com/prefix/TESTCCLW.executive.2.2.json"

or 

"download_url": "https://cdn.dev.climatepolicyradar.org/prefix/TESTCCLW.executive.2.2.json"